### ISSUE TITLE
Let's document: doctl

### ISSUE THREAD

**Issue by 258204** [tldr-pages/tldr#7475]
[] 1-click         
[]  account        
[]  apps           
[]  auth           
[]  balance        
[]  billing-history
[]  completion     
[]  compute        
[]  databases      
[]  help           
[]  invoice        
[]  kubernetes     
[]  monitoring     
[]  projects       
[]  registry       
[]  version        
[]  vpcs

**Comment by reinhart1010** [comment]
Oh it's DigitalOcean: https://docs.digitalocean.com/reference/doctl/

**Comment by cpetrella-sketch** [comment]
Can I try this?

**Comment by navarroaxel** [comment]
@cpetrella-sketch Yeah, you can send a PR for one command in the list. :rocket:

**Comment by 258204** [comment]
> Can I try this?

Yes absolutely! Be sure to link your PR in this thread so I can check it off the list.

**Comment by 258204** [comment]
The structure is a little more interesting than aws due to all the nested commands. The list in the initial issue doesn't fully illustrate the work necessary. 

https://docs.digitalocean.com/reference/doctl/reference/

I just realized how much money it's gonna cost to test out all these commands....

**Comment by theevildoof** [comment]
Can I work on this issue too?

**Comment by reinhart1010** [comment]
> Can I work on this issue too? 

Sure @theevildoof just make sure that there are no ongoing pull requests for these subcommands.

**Comment by Managor** [comment]
There needs to be a page for the base command.
